packages/plugin-browserify/src/plugin.ts

```diff
@@ -1,26 +1,28 @@
-import { Transform, TransformCallback } from 'stream';
-import { promises as fs } from 'fs';
-import os from 'os';
-import pathUtils from 'path';
-import { BrowserifyObject } from 'browserify';
 import {
   checkManifest,
+  dotenvSnap,
   evalBundle,
   postProcessBundle,
   PostProcessOptions,
 } from '@metamask/snap-utils';
+import { BrowserifyObject } from 'browserify';
 import { fromSource } from 'convert-source-map';
+import { promises as fs } from 'fs';
+import os from 'os';
+import pathUtils from 'path';
+import { Transform, TransformCallback } from 'stream';
 
 const TEMP_BUNDLE_PATH = pathUtils.join(os.tmpdir(), 'snaps-bundle.js');
 
 type PluginOptions = {
   eval?: boolean;
   manifestPath?: string;
   writeManifest?: boolean;
+  dotenv?: boolean;
```

Comment:
Maybe use something like `useEnvironmentVariables`, since not everyone may know what `dotenv` is, and it may cause confusion.